app/src/main/res/layout/activity_main_page.xml

```diff
@@ -0,0 +1,364 @@
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_alignParentBottom="true"
+            android:background="@color/light_grey">
+
+            <LinearLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/margin_image_inside_layout"
+                android:orientation="vertical">
+
+                <ImageButton
+                    android:id="@+id/profile"
+                    android:layout_width="@dimen/image_button_home_page"
+                    android:layout_height="@dimen/image_button_home_page"
+                    android:layout_gravity="center_horizontal"
+                    android:layout_marginTop="@dimen/margin_image_inside_layout"
+                    android:background="@drawable/profile"
+                    android:onClick="goToProfileActivity"
+                    android:contentDescription="@string/profile_button" />
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/scroll_view_margin_text"
+                    android:layout_gravity="center_horizontal"
+                    android:text="@string/profile"
+                    android:textColor="@color/dark_blue"
+                    android:textSize="@dimen/icon_font"
+                    android:textStyle="bold" />
+
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentEnd="true"
+                android:orientation="vertical"
+                tools:ignore="RelativeOverlap">
+
+                <ImageButton
+                    android:layout_width="@dimen/image_button_home_page"
+                    android:layout_height="@dimen/image_button_home_page"
+                    android:layout_gravity="center_horizontal"
+                    android:layout_marginTop="@dimen/margin_image_inside_layout"
+                    android:background="@drawable/medication"
+                    android:contentDescription="@string/skills_button"
+                    tools:ignore="ImageContrastCheck" />
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/scroll_view_margin_text"
+                    android:layout_gravity="center_horizontal"
+                    android:text="@string/skills"
+                    android:textColor="@color/dark_blue"
+                    android:textSize="@dimen/icon_font"
+                    android:textStyle="bold" />
+
+            </LinearLayout>
+        </RelativeLayout>
+
+        <ImageButton
+            android:id="@+id/HELP"
```

Comment:
The id could be a little more precise, given this is the name of the app it might be confusing later on.

eg:


```suggestion
            android:id="@+id/HELP_main_button"
```